src/Pact/Runtime/Capabilities.hs

```diff
@@ -276,3 +279,8 @@ checkSigCaps sigs = go
     match allowEmpty granted sigCaps =
       (S.null sigCaps && allowEmpty) ||
       not (S.null (S.intersection granted sigCaps))
+
+verifierIsInScope :: S.Set UserCapability -> Eval e Bool
```

Comment:
```suggestion
capabilitiesAcquired :: S.Set UserCapability -> Eval e Bool
```
or `capabilityAcquired'`, and pls move next to `capabilityAcquired`

("in scope" has multiple meanings, and in the capability codebase "acquired" is how we refer to capabilities brought into scope with `with-capability`)

Reply:
I started doing something else here. Now I check specifically that we are evaluating a cap that's in the verifier cap list. Check it out.